pullRequestReview.js

```diff
@@ -0,0 +1,70 @@
+var express = require('express');
+
+import { request, abort, } from 'request-promise';
+
+const port = process.env.PORT || 1337;
+
+const app = express();
+
+async function incoming2outgoingPayload(payload) {
+  import jsontoxml from 'jsontoxml';
+  const itemsToXML = {}
+  for(let i = 0; i <= payload.items.length; i++) {
+    const startDate = new Date(payload.items[i].start);
+    const endDate = new Date(payload.items[i].end);
+    itemsToXML[`vacation-${startDate.toString()}-${endDate.toString()}`] = payload.items[i].approved == true ? true : false;
```

Comment:
`payload.items[i].approved == true ? true : false`

`payload.items[i].approved` is already going to be `true` or `false` so no need for this ternary